### PR TITLE
Draft: Add dragonflight support

### LINKED ISSUE #1: Dragonflight support
Hi,

First, thank you very much for your addon. Really much appreciated and very useful. I would like to know if you would accept PR to support Dragonflight VOTI ? I think it just a matter of adding some data in [data.lua](/Data.lua) file. But as it is also adding a new extension key, maybe not. I've never developed a wow addon in the past, but I would be more than happy to work on the MR !

I've already looked for the VOTI quest on wowhead:
- Normal : [71018](https://www.wowhead.com/quest=71018/vault-of-the-incarnates-break-a-few-eggs)
- Heroic : [71019](https://www.wowhead.com/quest=71019/vault-of-the-incarnates-break-a-few-eggs#comments)
- Mythic : [71020](https://www.wowhead.com/quest=71020/vault-of-the-incarnates-break-a-few-eggs)

> Unlike some other raid skip quest in the past, unlocking the mythic difficulty does not grant skip for lower difficulty. I could not find any in game confirmation.

### PR DESCRIPTION
**Pull request by Skylli202**
Hi,

Did not wait your answer to make a PR, as I liked the exercise (never coded a wow addon). Feel free to reject it if you do not want any contribution.

This is only a draft, as the following items need to be taken care of:
- Both VOTI and Aberrus `instanceId` are wrong. Could not find what was the ID that you used on the other instance. Only found `zoneId` on wowhead, but did not match with the one you had for previous raid.
- Aberrus seems to only have two skip quests (the normal skip seems to be unavailable) [source](https://www.wowhead.com/quests/raids/aberrus-the-shadowed-crucible), I have no clue on much it can break the rest of the addon. From my in-game test it has no impact : 
image

> PS: I am sorry about auto format on save who edited some line who does not relate to this MR.

Closes GH-1

### CODE REVIEW

File: Data.lua [hunk 1]
```diff
@@ -63,6 +63,9 @@ RaidSkipper.raid_skip_quests = {
         raids = {
             -- instanceId is not correct, I do know which ID should be used.
             { name = "Vault of the Incarnates", instanceId = 2296, mythicId = 71020, heroicId = 71019, normalId = 71018 },
+            -- instanceId is not correct, I do know which ID should be used.
+            -- Aberrus seems to only have a heroic and mythic quest, no normal quest https://www.wowhead.com/zone=14663/aberrus-the-shadowed-crucible#quests
+            { name = "Aberrus, the Shadowed Crucible", instanceId = 2296, mythicId = 76086, heroicId = 76085 },
```
Comment (HarielMods) [inline Data.lua]: ATSC instaceId is 14663

File: Data.lua [hunk 2]
```diff
@@ -63,6 +63,9 @@ RaidSkipper.raid_skip_quests = {
         raids = {
             -- instanceId is not correct, I do know which ID should be used.
             { name = "Vault of the Incarnates", instanceId = 2296, mythicId = 71020, heroicId = 71019, normalId = 71018 },
```
Comment (HarielMods) [inline Data.lua]: VOTI instanceId is 14030

**Review by HarielMods — CHANGES_REQUESTED** [review]
Change those instanceIds and I'll get this PR approved!

File: Data.lua (hunk 1 above)
Comment (Skylli202) [reply Data.lua]: Correction done

File: Data.lua (hunk 2 above)
Comment (Skylli202) [reply Data.lua]: Correction done

**Comment by BS5** [comment]
Adding this pull request!

**Pull request merged**